reframe/core/pipeline.py

```diff
@@ -647,22 +647,36 @@ def pipeline_hooks(cls):
     #: :default: :class:`None`
     perf_patterns = variable(typ.Dict[str, _DeferredExpression], type(None))
 
-    #: Mapping with the performance variables of the test.
+    #: The performance variables associated with the test.
+    #:
+    #: In this context, a performance variable is a key-value pair, where the
+    #: key is the desired variable name and the value is the deferred
+    #: performance expression (i.e. the result of a :doc:`deferrable
```

Comment:
Here I suggest adding `.. _deferrable-perf-functions:` before the section in the sanity functions documentation. Then you can reference it anywhere with ``:ref:`deferrable-perf-functions` `` and the title of the section will be inserted. To avoid typing here, I would also change this to ``see :ref:`..` for more information`.